DOC-440 K8s guides for 'Developer Guides' (#1173)
RedisLabs/redislabs-docs: content/platforms/kubernetes/tasks/replica-of-k8s.md

---
Title: Creating replica databases on Kubernetes

> description: This task describes how to create and automate database replicas via the database controller

Reviewer: ```suggestion
description: How to create and automate database replicas using the database controller
```

weight: 42
alwaysopen: false
categories: ["Platforms"]
aliases:
---

> You can configure a replica of database by creating an item in

Reviewer: ```suggestion
You can configure a replica of a database by creating an item in
```

> Configuring a replica of database can be accomplished by creating an item in

Reviewer: Please use the active voice: https://developers.google.com/style/voice
